tests/test_tree.py

```diff
@@ -0,0 +1,19 @@
+import sklearn
+import numpy as np
+import shap
+
+def test_sklearn_interaction():
```

Comment:
There should probably be some kind of test here for the json loading in LightGBM. We can comment it if it makes travis unhappy.